src/utils.test.ts

```diff
@@ -0,0 +1,145 @@
+import { dataFrameFromJSON, DataFrameJSON } from '@grafana/data';
+
+import { frameToMetricFindValues } from './utils';
+
+describe('Test utils', () => {
+  it('convert simple values', () => {
+    const df = dataFrameFromJSON(listAssetsResponse as DataFrameJSON);
+    expect(frameToMetricFindValues(df)).toMatchInlineSnapshot(`
```

Comment:
hymm -- something funky here.  The result has 1 row, so I would expect 1 result 🤔